application/models/Cobrancas_model.php

```diff
@@ -36,11 +36,12 @@ public function get($table, $fields, $where = '', $perpage = 0, $start = 0, $one
 
     public function getById($id)
     {
-        $this->db->select('vendas.*, clientes.*, Cobrancas.*, clientes.email as emailCliente, lancamentos.data_vencimento, usuarios.telefone as telefone_usuario, usuarios.email as email_usuario, usuarios.nome');
+        $this->db->select('vendas.*, clientes.*, Cobrancas.*,os.* clientes.email as emailCliente, lancamentos.data_vencimento, usuarios.telefone as telefone_usuario, usuarios.email as email_usuario, usuarios.nome');
         $this->db->from('vendas');
         $this->db->join('clientes', 'clientes.idClientes = vendas.clientes_id');
         $this->db->join('usuarios', 'usuarios.idUsuarios = vendas.usuarios_id');
         $this->db->join('cobrancas', 'cobrancas.vendas_id = vendas.idVendas');
+        $this->db->join('cobrancas', 'cobrancas.os_id = os.isos');
```

Comment:
`         $this->db->join('cobrancas', 'cobrancas.os_id = os.isos'); `

Aqui não seria idOs?

`         $this->db->join('cobrancas', 'cobrancas.os_id = os.idOs'); `

Reply:
Erro ao digitar😔

Reply:
Tranquilo. Acontece